core/src/validator.rs

```diff
@@ -1043,8 +1043,13 @@ impl Validator {
             bank_forks.clone(),
             config.repair_whitelist.clone(),
         );
+        let (repair_quic_endpoint_sender, repair_quic_endpoint_receiver) = unbounded();
```

Comment:
If both sender and receiver are passed into ServeRepairService::new, why not initialize them in that function itself?

Reply:
`repair_quic_endpoint_sender` is also sent to client side of repair. see https://github.com/solana-labs/solana/pull/33016